sql/core/src/test/scala/org/apache/spark/sql/SparkSessionExtensionSuite.scala

```diff
@@ -52,6 +54,12 @@ class SparkSessionExtensionSuite extends SparkFunSuite {
     }
   }
 
+  test("SPARK-28292 inject analyzer hint") {
```

Comment:
For a new feature, we don't use JIRA ID in the test case. Please remove `SPARK-28292 ` prefix.

Reply:
Remove it , Thanks